SSHDriver: implement user switching via su (#1220)
labgrid-project/labgrid: doc/configuration.rst

    target.
  - explicit_sftp_mode (bool, default=False): if set to True, `put()` and `get()` will
    explicitly use the SFTP protocol for file transfers instead of scp's default protocol
  - su_username(str, default="root"): only used if su_password is set

>   - su_prompt(str, default="Passowrd:"): prompt string for su

Reviewer: ```suggestion
  - su_prompt(str, default="Password:"): prompt string for su
```

>   - su_password(str, default=None): su password for the user set via su_username

Reviewer: ```suggestion
  - su_password(str): optional, su password for the user set via `su_username`
```

.. note::

>    Using the su support will automatically enable stderr_merge, since ssh this

Reviewer: ```suggestion
   Using the su support will automatically enable `stderr_merge`, since ssh this
```
